web_external/js/views/widgets/StyleWmsDatasetWidget.js

```diff
@@ -48,7 +48,8 @@ minerva.views.StyleWmsDatasetWidget = minerva.View.extend({
                 this.dataset.fetch();
             }, this));
         },
-        'submit #m-wms-style-form': function (e) {
+
+        'submit #m-wms-point-style-form': function (e) {
```

Comment:
It looks like the submit handler from the old template (`#m-wms-style-form`) will no longer called.